docker/cyclus-deps/Dockerfile

```diff
@@ -31,19 +31,50 @@ RUN apt-get update && \
 # conda packages
 #
 RUN conda config --add channels conda-forge
+RUN conda update -n base -c defaults conda
 RUN conda update -y --all && \
-    conda install -y openssh gxx_linux-64 gcc_linux-64 cmake make docker-pycreds git xo python-json-logger \
-	                     python=3.6 glibmm glib=2.56 libxml2 libxmlpp libblas \
-	                     libcblas liblapack pkg-config coincbc=2.9 boost-cpp hdf5 \
-	                     sqlite pcre gettext bzip2 xz setuptools nose pytables \
-	                     pandas jinja2 cython==0.26 websockets pprintpp &&\
+    conda install -y \
```

Comment:
Does this list of packages inform any necessary changes to the installation instructions ? (are there new deps? old deps that aren't needed?)